tools/lambda-promtail/main.tf

```diff
@@ -34,11 +34,27 @@ resource "aws_iam_role_policy" "logs" {
         ],
         "Effect" : "Allow",
         "Resource" : "arn:aws:logs:*:*:*",
+      },
+      {
+        "Action" : [
+          "s3:GetObject",
+        ],
+        "Effect" : "Allow",
+        "Resource" : "arn:aws:s3:::*/*",
```

Comment:
If it's possible to do this for the bucket names we should ideally try to do the same for the log groups in the above ```"Resource" : "arn:aws:logs:*:*:*",```, but I don't know how that would work out given the lambda also creates it's own log group and log events. We could do this as a follow up PR.

Reply:
For logs: since we are creating the lambda we know the name of the log group, but by default AWS gives those 3 permissions on every log group

For S3: will fix it

Reply:
fixed s3 permissions, will try to update docs later today/tomorrow